lib/puppet/type/ec2_autoscalinggroup.rb

```diff
@@ -62,6 +62,13 @@ def insync?(is)
     end
   end
 
+  newproperty(:load_balancers_names, :array_matching => :all) do
```

Comment:
I think this should be called `load_balancers` to match the naming convention elsewhere, Also, you'll want to set an `autorequire` to point to the `elb_loadbalancer` type, in order for Puppet to know that it should create/act on the load balancer before the autoscaling group if both are described in Puppet.